docs/UserGuide.adoc

```diff
@@ -223,11 +237,15 @@ The `redo` command fails as there are no `undo` commands executed previously.
 Clears all entries from the address book. +
 Format: `clear`
 
+Alias Format: `c`
+
 === Exiting the program : `exit`
 
 Exits the program. +
 Format: `exit`
 
+Alias Format: `x`
+
 === Saving the data
```

Comment:
Might want to update the `Command Summary` below so that the existence of aliases is made known

Reply:
updated